.env.example

```
@@ -27,6 +27,11 @@ DJANGO_SECRET_KEY=change-me-in-production
 CELERY_BROKER_URL=redis://redis:6379/0
 CELERY_RESULT_BACKEND=redis://redis:6379/0
 
+# --- Neo4j ---
+NEO4J_PASSWORD=swh_neo4j_dev
+NEO4J_PAGECACHE=512m
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEO4J_PAGECACHE key should go before the NEO4J_PASSWORD key

(UnorderedKey)